pkg/stanza/internal/fileconsumer/file.go

```diff
@@ -25,25 +25,25 @@ import (
 
 	"go.uber.org/zap"
 
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/entry"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/helper"
 )
 
-// Input is an operator that monitors files for entries
+type EmitFunc func(ctx context.Context, attrs *FileAttributes, token []byte)
+
+// TODO rename this struct
```

Comment:
Is this TODO for the next PR or do we need an issue to track it?

Reply:
Yes, these are meant to be be very temporary. I plan to iterate over this codebase a bit yet, so won't miss them.